From 2e60e3972ee8dadfebace50255c9275feae4dc6d Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Sebastian=20Dr=C3=B6ge?= <sebastian@centricular.com>
Date: Tue, 21 Oct 2014 13:46:18 +0200
Subject: [PATCH] Disable building of the tests

We don't run them anyway and there are build system quirks in there.
---
 Makefile.am | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index 03fa352..7e11d24 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -18,10 +18,6 @@ ACLOCAL_AMFLAGS = -I build ${ACLOCAL_FLAGS}
 
 DIST_SUBDIRS = src doc util boilerplate test perf
 SUBDIRS = src doc util
-# libpng is required for our test programs
-if CAIRO_HAS_PNG_FUNCTIONS
-SUBDIRS += boilerplate test perf
-endif
 
 configure: cairo-version.h
 
-- 
2.1.1

